src/main/java/com/minecolonies/coremod/commands/killcommands/BarbarianKillCommand.java

```diff
@@ -1,5 +1,6 @@
 package com.minecolonies.coremod.commands.killcommands;
 
+import com.minecolonies.coremod.commands.IActionCommand;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.minecolonies.coremod.commands.IActionCommand'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)